Sources/Mini/StateType.swift

```diff
@@ -1,28 +0,0 @@
-/*
- Copyright [2019] [BQ]
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
- http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
- */
-
-import Foundation
-
-public protocol StateType {
```

Comment:
Why deleting this protocol?

Reply:
I have not seen it necessary at any point in the flow, in fact I could delete it completely but I leave it in case in Rx it makes more sense.